apps/src/dance/lab2/views/GenerateDancer.tsx

```diff
@@ -158,6 +151,86 @@ const GenerateDancer: React.FunctionComponent<DancerGenerateProps> = ({
     setAiGenerateState('done');
   }, [generateDancerCache]);
 
+  const canvasRef = useRef<HTMLCanvasElement | null>(null);
```

Comment:
Curious whether this dancer rendering code might make sense in a standalone file.  In fact, could the new `DancerOverlayCanvas` be used here?